policyengine_us/parameters/gov/states/ri/tax/income/agi/subtractions/social_security/limit/income.yaml

```diff
@@ -13,13 +13,16 @@ metadata:
   # The tax form points to this worksheet: Social Security modification on Schedule M, line 1t.
   - title: Rhode Island 2022 Modification for Taxable social security income worksheet
     href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/Social%20Security%20Worksheet_w.pdf#page=1
+  - title: Rhode Island 2023 Modification for Taxable social security income worksheet
```

Comment:
I think the amounts here are adjusted with inflation:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/c2bca45b-ddcb-472b-9c5e-dbb3a1434a97)

We need to add some metadata for inflation adjustments and rounding, see DC:
https://github.com/PolicyEngine/policyengine-us/pull/4306